### ISSUE TITLE
Checkbox is not checked using value attribute

### ISSUE THREAD

**Issue by cschroeter** [platosha/angular-polymer#45]
``` typescript
import { Component } from '@angular/core';
import { PolymerElement } from '@vaadin/angular2-polymer';

@Component({
  moduleId: module.id,
  selector: 'app-root',
  templateUrl: 'app.component.html',
  styleUrls: ['app.component.css'],
  directives: [
    PolymerElement('paper-checkbox'),
    PolymerElement('paper-button'),
  ]
})
export class AppComponent {

  public test: boolean = true;

}
```

``` html
<p>
  Value of test: {{test}}
  <paper-checkbox [(value)]="test">Template</paper-checkbox>
</p>
```

<img width="257" alt="bildschirmfoto 2016-07-15 um 00 26 20" src="https://cloud.githubusercontent.com/assets/1846056/16857831/c8111382-4a22-11e6-9aed-074f826578a2.png">

**Comment by tallaxes** [comment]
Should be `checked` instead of `value`:

``` html
<p>
  Value of test: {{test}}
  <paper-checkbox [(checked)]="test">Template</paper-checkbox>
</p>
```

**Comment by cschroeter** [comment]
That does the trick but then you still can not use the ngControl directive for model driven forms

``` html
<p>{{form.value | json}}</p>

<form [ngFormModel]="form">
  <paper-input label="Name" ngControl="name" error-message="Max 10 chars"></paper-input>
  <paper-checkbox ngControl="classic">Classic</paper-checkbox>
</form>
```

``` typescript
import { Component } from '@angular/core';
import { ControlGroup, FormBuilder, Validators } from '@angular/common';
import { PolymerElement } from '@vaadin/angular2-polymer';

@Component({
  moduleId: module.id,
  selector: 'app-root',
  templateUrl: 'app.component.html',
  styleUrls: ['app.component.css'],
  directives: [
    PolymerElement('paper-button'),
    PolymerElement('paper-checkbox'),
    PolymerElement('paper-input'),
  ]
})
export class AppComponent {

  private form: ControlGroup;

  private car = {
    name: '',
    classic: true
  };

  constructor(private fb: FormBuilder) {
  }

  public ngOnInit(): void {
    this.form = this.fb.group({
      name: [this.car.name, Validators.maxLength(10)],
      classic: [this.car.classic]
    });
  }

}
```

**Comment by Saulis** [comment]
Added a feature to target `ngControl` to use `checked` instead of `value` property for elements which extend `IronCheckedElementBehavior`

**Comment by platosha** [comment]
Fixed in #62
